packages/email-field/README.md

```diff
@@ -1,5 +1,50 @@
 # @vaadin/email-field
 
-> ⚠️ Work in progress, please do not use this component yet.
+An extension of `vaadin-text-field` component that only accepts email addresses as input.
 
-The new version of `vaadin-email-field` component using slotted `<input>`.
+[Live Demo ↗](https://vaadin.com/docs/latest/ds/components/email-field)
+
+```html
+<vaadin-email-field label="Email"></vaadin-email-field>
+```
+
+## Installation
+
+Install the component:
+
+```sh
+npm i @vaadin/email-field --save
+```
+
+Once installed, import the component in your application:
+
+```js
+import '@vaadin/email-field';
+```
+
+## Getting started
+
+Vaadin components use the Lumo theme by default.
+
+To use the Material theme, import the component from the `theme/material` folder.
```

Comment:
Wouldn't it be better to tell user exactly what to import? I mean is it like this?
```js
import '@vaadin/lumo/email-field';
````

or this:
```js
import '@vaadin/theme/lumo/email-field';
````

Reply:
Good point. Updated the PR with code snippets as you suggested.